internal/resource/list.go

```diff
@@ -299,11 +306,11 @@ func (l *list) Reconcile(informer *wa.Informer, path *string) error {
 		LabelSelector: l.labelSelector,
 		FieldSelector: l.fieldSelector,
 	}
-
-	if items, err = l.resource.List(l.namespace, opts); err != nil {
+	if items, err := l.resource.List(l.namespace, opts); err == nil {
```

Comment:
I prefer when the indentations levels of `ifs` means error and the first identation level is the "normal" path of the coding, like:
```go
items, err := l.resource.List(l.namespace, opts)
if err != nil {
	// the insides of `ifs` means that something is wrong.
	return err
}

l.update(items)
return nil
```

If everything goes correct, you can simply read the left most statements.

That is a personal opinion, change if you agree :)

Reply:
Good catch. Agreed!!